canonical/test_observer
# Fix fetching snaps family returns 500

Resolves [RTW-144](https://warthogs.atlassian.net/browse/RTW-144)

This PR includes:
- Actual bug fix through refactor
  - Add custom property to Stage ORM named `latest_artefacts` that gets the latest artefacts
  - Sort stages from Family ORM directly
  - Make StageDTO map `latest_artefacts` to `artefact` key
- Update some dependencies
- Add repr to data models

I've already deployed these changes to staging to make sure they work.

[RTW-144]: https://warthogs.atlassian.net/browse/RTW-144?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
